resources/provider/provider.go

```diff
@@ -100,6 +100,7 @@ func Provider() *provider.Provider {
 			"storage.accounts":                     storage.StorageAccounts(),
 			"streamanalytics.jobs":                 streamanalytics.StreamanalyticsJobs(),
 			"subscription.subscriptions":           subscription.SubscriptionSubscriptions(),
+			"subscription.tenants":                 subscription.Tenants(),
```

Comment:
What is the relationship between tenant and subscriptions? Is their any hierarchy between the two?